components/installer/pkg/overrides/overrides.go

```diff
@@ -0,0 +1,115 @@
+package overrides
+
+import "k8s.io/client-go/kubernetes"
+
+type Provider struct {
+	common     Map
+	components map[string]Map
+}
+
+//Common returns overrides common for all components
+func (o *Provider) Common() Map {
+	res := o.common
+
+	if res == nil {
+		return Map{}
+	}
+
+	return res
+}
+
+//ForComponent returns overrides defined only for specified component
+func (o *Provider) ForComponent(componentName string) Map {
+
+	res := o.components[componentName]
+
+	if res == nil {
+		return Map{}
+	}
+
+	return res
+}
+
+//New returns new Data instance.
```

Comment:
If you use here` kubernetes.Interface ` instead of `kubernetes.Clientset` it will be easier for you to write tests for this component. You will not need working k8s for it